### PR TITLE
[ONNX] Fix an issue with #5755 and add Batch norm unit tests.

### PR DESCRIPTION
**Pull request by mbrookhart**
@masahi

@jwfromm discovered an issue with #5755 , infer_value chokes on ops with multiple outputs. I already knew this for Tuple, this adds a work around to infer batch_norm with the following TupleGetItem node instead of by itself.

In adding a regression test, I discovered that the Onnx importer for batch norm wasn't tested, so I added a unit test for it.

Are you guys aware of any other multi-output ops that might suffer the same problem?

### CODE REVIEW

**Comment by jwfromm** [comment]
thanks for the fix, LGTM.
